docs/controls/Chip.md

```diff
@@ -0,0 +1,85 @@
+# Chip
+
+## Summary
+
+`Chip` is derived `ToggleButton` that can be used for selection, filters or trigger action selection. `ChipGroup` can be used to display a list of `Chip`. 
```

Comment:
Missing a word I think! `Chip` is derived `ToggleButton` that

Reply:
Yeah this didn't make sense at all. I reworded this sentence,